src/core/library.c

```diff
@@ -829,6 +829,11 @@ QuicLibrarySetGlobalParam(
 
         MsQuicLib.Settings.RetryMemoryLimit = *(uint16_t*)Buffer;
         MsQuicLib.Settings.IsSet.RetryMemoryLimit = TRUE;
+
+        MsQuicLib.HandshakeMemoryLimit =
+            (MsQuicLib.Settings.RetryMemoryLimit * CxPlatTotalMemory) / UINT16_MAX;
+        QuicLibraryEvaluateSendRetryState();
```

Comment:
Can we put this _after_ the log below please? It might help with debugging in the future.